packages/client/e2e/step-definitions/setup.ts

```diff
@@ -6,14 +6,15 @@ const FLAGD_NAME = 'flagd-web';
 
 // register the flagd provider before the tests.
 console.log('Setting flagd web provider...');
-OpenFeature.setProvider(
-  new FlagdWebProvider({
-    host: 'localhost',
-    port: 8013,
-    tls: false,
-    maxRetries: -1,
-  })
-);
+const provider = new FlagdWebProvider({
+  host: 'localhost',
+  port: 8013,
+  tls: false,
+  maxRetries: -1,
+});
+// TODO: remove this when provider updated to properly use state.
```

Comment:
Please create an issue for this so we don't forget about it 😄

Reply:
OK, ya. I there's a couple other minor change here too, we can get rid of some set-timeouts now that we have init.

Reply:
https://github.com/open-feature/js-sdk/issues/518

Reply:
Don't forget to remove this comment.

Reply:
I've removed this comment _and_ removed all the `setTimeouts` in our test suite. Everything is working "properly" now with plain events, since all flagd providers are fully updated.